README.md

```diff
@@ -254,7 +256,7 @@ If you like this project...
 
```

Comment:
![Capture](https://user-images.githubusercontent.com/93039359/211852311-c396fe8b-4153-4f34-b44d-0ef96ca187b5.PNG)

- [ ] You didn't fill this section of your readme 😨 Kindly add the instructions here of how to start your application. 👍